### PR TITLE
[Bugfix/#2329] Change UBS certificate functionality

### PR DESCRIPTION
**Pull request by Edw1x**
# Activated certificate for 600 uah
image
image
image

# Activated certificate for 1000 uah
image
image

# Used certificate
image

# Expired certificate
image

# Note: if we have 0 uah to pay when we press order on 3 step, Fondy doesn't open a modal window.

### CODE REVIEW

File: src/assets/i18n/ubs/ru.json [hunk 1]
```diff
@@ -29,11 +29,10 @@
     "cancel": "Отменить",
     "failed-certificate": "Сертификат не принят, попробуйте еще раз",
     "not-found-certificate": "Сертификат не найден; проверьте правильность данных.",
-    "activated-certificate1": "Сертификат на сумму",
-    "activated-certificate2": "грн активировано, что превышает сумму вашего заказа. Остаток в размере",
-    "activated-certificate3": "грн будет начислено на Ваш бонусный счет для следующего заказа.",
-    "activated-certificate4": "грн активирован. Срок действия сертификата - до",
-    "activated-certificate5": "Сертификат уже использовано. Срок действия сертификата - до",
+    "activated-oversum": "Сертификат на сумму {{certificateSum}} грн включена, что превышает сумму вашего заказа. Остаток в размере {{certificateLeft}} грн будет начислено на Ваш бонусный счет для следующего заказа. Срок действия сертификата - до {{ certDate}} ",
```
Comment (AlekOK) [inline src/assets/i18n/ubs/ru.json:32]: "Сертификат на сумму {{certificateSum}} грн включенА

**Review by JimHolman1199 — APPROVED** [review]

**Review by roksolanadutchak — APPROVED** [review]

File: src/assets/i18n/ubs/ru.json (hunk 1 above)
Comment (JimHolman1199) [reply src/assets/i18n/ubs/ru.json:32]: agree

Comment (JimHolman1199) [reply src/assets/i18n/ubs/ru.json:32]: agree

Comment (Edw1x) [reply src/assets/i18n/ubs/ru.json]: image

**Review by roksolanadutchak — APPROVED** [review]

**Review by flypsoff — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

**Review by oliachipko — APPROVED** [review]

**Pull request merged**